cmd/root/install/mssql/server.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package mssql
```

Comment:
>e mssql [](http://example.com/codeflow?start=6&length=7)

this package name could be confusing if some .go file ever needs to reference both this one and the mssql package in github.com/microsoft/go-mssqldb

Reply:
Each package in this folder structure is the name of the command line, so this is the .go file/package for `sqlcmd install mssql`.  This is the top most package in the architecture, so no-one will use this package.  But it does take a dependency on the /internal/helpers/mssql package, which is wrapping pkg/sqlcmd for error handling and tracing when we connect to sql to disable the sa account.   This is a lot of packages named mssql, let me think about it.  I think if there are no layering violations there will never be more than one needed in scope.